static/src/stylesheets/module/onward/_rich-links-enhanced.scss

```diff
@@ -51,22 +38,16 @@
             opacity: .8;
         }
     }
-
 }
 
 .rich-link__byline {
     @include fs-header(3);
     padding-right: $gs-gutter;
 }
 
-.rich-link__title {
-    padding: 0;
-}
-
-.rich-link .rich-link__header {
-    padding: $gs-baseline / 3 $gs-gutter / 4 $gs-baseline;
-    box-sizing: border-box;
-    min-height: $gs-baseline * 3;
+.rich-link__byline {
```

Comment:
Is this a duplication of  [L57](https://github.com/guardian/frontend/pull/14118/files#diff-bbc8a915c2b5cac7c83501927eb823dcL57) ?